auth-service/src/main/resources/db/migration/V2__password_tokens.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE set_password_tokens
+(
+    user_cn    text UNIQUE PRIMARY KEY,
```

Comment:
```suggestion
    user_cn    text      PRIMARY KEY,
```
primary key implies unique